test/modellib/ModelLib.hpp

```diff
@@ -119,6 +120,14 @@ class ModelLibBuilder {
   onnx_mlir::ExecutionSession *exec;
 };
 
+// Padding schemes for Convolutions.
+#define AUTO_PAD_NOTSET 0
```

Comment:
Does it make sense here to have a `enum` or `enum class` ?

Reply:
ok, transforming to a enum